src/mgmt/mgmt.pb-c.c

```diff
@@ -515,15 +515,17 @@ const ProtobufCMessageDescriptor mgmt__get_attach_info_resp__descriptor =
   (ProtobufCMessageInit) mgmt__get_attach_info_resp__init,
   NULL,NULL,NULL    /* reserved[123] */
 };
-static const ProtobufCMethodDescriptor mgmt__mgmt_svc__method_descriptors[3] =
+static const ProtobufCMethodDescriptor mgmt__mgmt_svc__method_descriptors[4] =
 {
   { "Join", &mgmt__join_req__descriptor, &mgmt__join_resp__descriptor },
   { "CreatePool", &mgmt__create_pool_req__descriptor, &mgmt__create_pool_resp__descriptor },
+  { "DestroyPool", &mgmt__destroy_pool_req__descriptor, &mgmt__destroy_pool_resp__descriptor },
```

Comment:
Is this supposed to be here? It looks like you're adding a new dRPC mgmt call in a PR thats supposed to be changing the error code handling?

Reply:
it's just a change in the response name, as mentioned in the commit message